tutorials/api-mgmt-isuite-initial-setup/api-mgmt-isuite-initial-setup.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Setting Up API Management from Integration Suite
```

Comment:
Use a more active verb, like "Set Up..."

Reply:
I have changed the title to "Set Up API Management from Integration Suite".